packages/theme-dark-extension/style/variables.css

```
@@ -167,7 +167,7 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-content-font-color2: rgba(255, 255, 255, 0.7);
   --jp-content-font-color3: rgba(255, 255, 255, 0.5);
   --jp-content-link-color: var(--md-blue-300);
-  --jp-content-link-visited-color: rgba(255, 255, 255, 0.3);
+  --jp-content-link-visited-color: var(--md-purple-400);
```

This fails WCAG AA. What about:

```suggestion
  --jp-content-link-visited-color: var(--md-purple-300);
```